.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  goreleaser:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - name: Setup Snapcraft
+        run: |
+          sudo snap install snapcraft --classic
+          snapcraft login --with - <<<"${SNAP_TOKEN}"
+        env:
+          SNAP_TOKEN: ${{ secrets.SNAP_LOGIN }}
+          SNAPCRAFT_BUILD_ENVIRONMENT: host
```

Comment:
We decided in today's community meeting that we would go with only the release binaries in the GitHub release and target external package managers in the next iteration. Based on that, I guess this can go away.

Reply:
we can be confident, but, we need to create the required accounts into the different package manager repositories. And someone with the sufficient privileges need to put that in our repository configuration.

Reply:
@SaschaSchwarze0 do you have permission to do that?